<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EnlargeParamSelectWidget</class>
 <widget class="QWidget" name="EnlargeParamSelectWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>310</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>470</width>
       <height>272</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2" stretch="6,84">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="widget_3" native="true">
        <property name="styleSheet">
         <string notr="true">#widget_3 {
background-color: #4586E4;
}</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="leftMargin">
          <number>19</number>
         </property>
         <item>
          <widget class="QLabel" name="label">
           <property name="styleSheet">
            <string notr="true">#label {

font-family: Microsoft YaHei;
font-size: 14px;
color: #FFF;
}</string>
           </property>
           <property name="text">
            <string>放大设置</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>802</width>
             <height>17</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QToolButton" name="close_btn">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <property name="styleSheet">
         <string notr="true">#widget_2 {
background-color: #FFF;
}
</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QWidget" name="switch_widget" native="true">
           <property name="styleSheet">
            <string notr="true">QRadioButton {
font-family: Microsoft YaHei;
font-size: 13px;
color: #444444;
}</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_4" stretch="4,1">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QWidget" name="top_widget" native="true">
              <property name="styleSheet">
               <string notr="true">QLabel {
font-family: Microsoft YaHei;
font-size: 13px;
color: #444444;
}

QRadioButton {
font: 13px Microsoft YaHei;
color: #444444;
}</string>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="1,4">
               <property name="spacing">
                <number>0</number>
               </property>
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QWidget" name="left_widget" native="true">
                 <layout class="QVBoxLayout" name="verticalLayout" stretch="1,2,4,2,3">
                  <property name="spacing">
                   <number>0</number>
                  </property>
                  <property name="leftMargin">
                   <number>0</number>
                  </property>
                  <property name="topMargin">
                   <number>0</number>
                  </property>
                  <property name="rightMargin">
                   <number>0</number>
                  </property>
                  <property name="bottomMargin">
                   <number>0</number>
                  </property>
                  <item>
                   <spacer name="verticalSpacer">
                    <property name="orientation">
                     <enum>Qt::Vertical</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>20</width>
                      <height>40</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                  <item>
                   <widget class="QLabel" name="label_2">
                    <property name="styleSheet">
                     <string notr="true"/>
                    </property>
                    <property name="text">
                     <string>图片类型:</string>
                    </property>
                    <property name="alignment">
                     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QWidget" name="widget_5" native="true">
                    <layout class="QVBoxLayout" name="verticalLayout_8">
                     <property name="spacing">
                      <number>0</number>
                     </property>
                     <property name="rightMargin">
                      <number>0</number>
                     </property>
                     <property name="bottomMargin">
                      <number>0</number>
                     </property>
                     <item>
                      <widget class="QLabel" name="label_3">
                       <property name="styleSheet">
                        <string notr="true">#label_3 {

}</string>
                       </property>
                       <property name="text">
                        <string>放大倍数:</string>
                       </property>
                       <property name="alignment">
                        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <spacer name="verticalSpacer_5">
                       <property name="orientation">
                        <enum>Qt::Vertical</enum>
                       </property>
                       <property name="sizeHint" stdset="0">
                        <size>
                         <width>20</width>
                         <height>40</height>
                        </size>
                       </property>
                      </spacer>
                     </item>
                    </layout>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="label_4">
                    <property name="styleSheet">
                     <string notr="true"/>
                    </property>
                    <property name="text">
                     <string>降噪程度:</string>
                    </property>
                    <property name="alignment">
                     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <spacer name="verticalSpacer_2">
                    <property name="orientation">
                     <enum>Qt::Vertical</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>20</width>
                      <height>40</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <widget class="QWidget" name="right_widget" native="true">
                 <layout class="QVBoxLayout" name="verticalLayout_3" stretch="1,2,4,2,3">
                  <property name="spacing">
                   <number>0</number>
                  </property>
                  <property name="leftMargin">
                   <number>12</number>
                  </property>
                  <property name="topMargin">
                   <number>0</number>
                  </property>
                  <property name="rightMargin">
                   <number>0</number>
                  </property>
                  <property name="bottomMargin">
                   <number>0</number>
                  </property>
                  <item>
                   <spacer name="verticalSpacer_4">
                    <property name="orientation">
                     <enum>Qt::Vertical</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>20</width>
                      <height>40</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                  <item>
                   <widget class="QWidget" name="type_widget" native="true">
                    <property name="styleSheet">
                     <string notr="true"/>
                    </property>
                    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="3,3,4">
                     <property name="leftMargin">
                      <number>16</number>
                     </property>
                     <property name="topMargin">
                      <number>0</number>
                     </property>
                     <property name="bottomMargin">
                      <number>0</number>
                     </property>
                     <item>
                      <widget class="QRadioButton" name="type_radio_carton">
                       <property name="text">
                        <string>卡通/插图</string>
                       </property>
                       <property name="checked">
                        <bool>true</bool>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QRadioButton" name="type_radio_pic">
                       <property name="text">
                        <string>照片</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <spacer name="horizontalSpacer_3">
                       <property name="orientation">
                        <enum>Qt::Horizontal</enum>
                       </property>
                       <property name="sizeHint" stdset="0">
                        <size>
                         <width>40</width>
                         <height>20</height>
                        </size>
                       </property>
                      </spacer>
                     </item>
                    </layout>
                   </widget>
                  </item>
                  <item>
                   <widget class="QWidget" name="magnification_widget" native="true">
                    <property name="styleSheet">
                     <string notr="true">#magnification_widget {

}</string>
                    </property>
                    <layout class="QHBoxLayout" name="horizontalLayout_7">
                     <property name="spacing">
                      <number>0</number>
                     </property>
                     <property name="leftMargin">
                      <number>0</number>
                     </property>
                     <property name="topMargin">
                      <number>0</number>
                     </property>
                     <property name="rightMargin">
                      <number>0</number>
                     </property>
                     <property name="bottomMargin">
                      <number>0</number>
                     </property>
                     <item>
                      <layout class="QVBoxLayout" name="verticalLayout_7" stretch="1,4">
                       <property name="spacing">
                        <number>0</number>
                       </property>
                       <item>
                        <widget class="QWidget" name="widget_4" native="true">
                         <property name="styleSheet">
                          <string notr="true">#widget_4 {
padding-left:16px;
}</string>
                         </property>
                         <layout class="QHBoxLayout" name="horizontalLayout_3">
                          <property name="leftMargin">
                           <number>16</number>
                          </property>
                          <property name="topMargin">
                           <number>9</number>
                          </property>
                          <property name="bottomMargin">
                           <number>0</number>
                          </property>
                          <item>
                           <widget class="QRadioButton" name="magn_radio_2">
                            <property name="text">
                             <string>2x</string>
                            </property>
                            <property name="checked">
                             <bool>true</bool>
                            </property>
                           </widget>
                          </item>
                          <item>
                           <widget class="QRadioButton" name="magn_radio_4">
                            <property name="text">
                             <string>4x</string>
                            </property>
                           </widget>
                          </item>
                          <item>
                           <widget class="QRadioButton" name="magn_radio_8">
                            <property name="text">
                             <string>8x</string>
                            </property>
                           </widget>
                          </item>
                          <item>
                           <widget class="QRadioButton" name="magn_radio_16">
                            <property name="text">
                             <string>16x</string>
                            </property>
                           </widget>
                          </item>
                          <item>
                           <widget class="QRadioButton" name="magn_radio_32">
                            <property name="text">
                             <string>32x</string>
                            </property>
                           </widget>
                          </item>
                         </layout>
                        </widget>
                       </item>
                       <item>
                        <layout class="QVBoxLayout" name="verticalLayout_6" stretch="0">
                         <property name="spacing">
                          <number>0</number>
                         </property>
                         <item>
                          <widget class="QLabel" name="label_5">
                           <property name="styleSheet">
                            <string notr="true">#label_5 {
font: 10px Microsoft YaHei;
color: #FF8888;
padding-left: 16px;
}</string>
                           </property>
                           <property name="text">
                            <string>升级后可放大8倍/16倍/32倍</string>
                           </property>
                           <property name="alignment">
                            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
                           </property>
                           <property name="indent">
                            <number>0</number>
                           </property>
                          </widget>
                         </item>
                        </layout>
                       </item>
                      </layout>
                     </item>
                     <item>
                      <spacer name="horizontalSpacer_4">
                       <property name="orientation">
                        <enum>Qt::Horizontal</enum>
                       </property>
                       <property name="sizeHint" stdset="0">
                        <size>
                         <width>104</width>
                         <height>20</height>
                        </size>
                       </property>
                      </spacer>
                     </item>
                    </layout>
                   </widget>
                  </item>
                  <item>
                   <widget class="QWidget" name="quality_widget" native="true">
                    <property name="styleSheet">
                     <string notr="true"/>
                    </property>
                    <layout class="QHBoxLayout" name="horizontalLayout_4">
                     <property name="leftMargin">
                      <number>16</number>
                     </property>
                     <property name="topMargin">
                      <number>0</number>
                     </property>
                     <property name="bottomMargin">
                      <number>0</number>
                     </property>
                     <item>
                      <widget class="QRadioButton" name="quality_radio_4">
                       <property name="text">
                        <string>无</string>
                       </property>
                       <property name="checked">
                        <bool>true</bool>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QRadioButton" name="quality_radio_1">
                       <property name="text">
                        <string>低</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QRadioButton" name="quality_radio_2">
                       <property name="text">
                        <string>中</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QRadioButton" name="quality_radio_3">
                       <property name="text">
                        <string>高</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QRadioButton" name="quality_radio_5">
                       <property name="text">
                        <string>最高</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <spacer name="horizontalSpacer_5">
                       <property name="orientation">
                        <enum>Qt::Horizontal</enum>
                       </property>
                       <property name="sizeHint" stdset="0">
                        <size>
                         <width>40</width>
                         <height>20</height>
                        </size>
                       </property>
                      </spacer>
                     </item>
                    </layout>
                   </widget>
                  </item>
                  <item>
                   <spacer name="verticalSpacer_3">
                    <property name="orientation">
                     <enum>Qt::Vertical</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>20</width>
                      <height>40</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                 </layout>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QWidget" name="bottom_widget" native="true">
              <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="23,0,0,2">
               <property name="spacing">
                <number>10</number>
               </property>
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>20</number>
               </property>
               <item>
                <spacer name="horizontalSpacer_2">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QPushButton" name="confirm_btn">
                 <property name="text">
                  <string>确认</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="cancel_btn">
                 <property name="text">
                  <string>取消</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer_6">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
